platform/solr/solr-factory/src/main/java/org/codice/solr/factory/impl/HttpSolrClientFactory.java

```diff
@@ -59,12 +66,13 @@
  *   <li>https.protocols: Secure protocols supported by the Solr server
  *   <li>https.cipherSuites: Cipher suites supported by the Solr server
  * </ul>
+ *
+ * @deprecated This class may be removed in the future
```

Comment:
I had a hard time seeing what was going on in this class. I did some refactoring with IntelliJ's refactoring tools to "drain the swamp".